setup.py

```diff
@@ -667,6 +667,17 @@ def make_relative_rpath(path):
                 sources=[]),
         )
 
+    # This is a placeholder while Nikita and I figure out how to add this to CMake.
```

Comment:
Since you don't need to actually build valgrind as add a dep, it should be as simple as adding the correct directory in the third party valgrind via `(target_)include_directories`

Reply:
PyBind is being a pest.

Reply:
If pybind gives you headache - you could also just build a `.so` and open it directly with `ctypes.CDLL`. Given that your functions have no args and return a bool at most, it'd be very simple. But in general pybind is nice and header-only

Reply:
Battered and broken, I've decided to just throw in the towel and drop it into `Modules.cpp`. In part because I want to make the 1.7 release, and in part because if I spend another moment with CMake I'll likely be driven to violence. For back-testing I'll probably just make a branch that make the independent `.so` and only has to work on my machine.